mcmd/commands/add.py

```diff
@@ -72,6 +72,29 @@ def arguments(subparsers):
                              type=str,
                              help="The token")
 
+    p_add_theme = p_add_subparsers.add_parser('theme',
+                                              help='Upload a bootstrap theme')
+    p_add_theme.set_defaults(func=add_theme,
+                             write_to_history=True)
+    p_add_theme.add_argument('bs3',
+                             type=str,
+                             metavar='Bootstrap 3 theme',
+                             help="The bootstrap 3 css theme file")
+    p_add_theme.add_argument('--bs4',
+                             type=str,
+                             metavar='Bootstrap 4 theme',
+                             help="The bootstrap4 css theme file (when not specified, the default molgenis theme will "
+                                  "be applied on bootstrap4 pages")
+
+    p_add_logo = p_add_subparsers.add_parser('logo',
+                                             help='Upload a logo to be placed on the left top of the menu')
+    p_add_logo.set_defaults(func=add_logo,
+                            write_to_history=True)
+    p_add_logo.add_argument('logo',
+                            type=str,
+                            metavar='LOGO IMAGE',
+                            help="The image you want to use as logo")
+
```

Comment:
We've discussed it in person already but I'll summarise it here:

Working with files should, by default, work like it works with the import command. So we'll add the following property to the config file:

```yaml
resources:
  dataset_folders: []
  resource_folders: []
```

All commands that upload/import non-dataset files should use these resource folders, unless specified otherwise:

```
mcmd add theme example.css                          <-- should look in the resource folders
mcmd add theme --from-path /my/path/to/example.css  <-- looks at the path
```